frontend/src/views/WheresMyGene/components/HeatMap/components/YAxisChart/index.tsx

```diff
@@ -125,8 +131,25 @@ const CellTypeButton = ({
   return (
     <FlexRowJustified data-test-id="cell-type-label-count">
       <FlexRow>
-        <CellTypeLabelStyle data-test-id="cell-type-name">
-          {name}
+        <CellTypeLabelStyle>
+          <Tooltip
+            title={
+              // Set tooltip content only if name is truncated
+              formattedName.includes("...") ? (
+                <CellTypeLabelTooltipStyle>{name}</CellTypeLabelTooltipStyle>
+              ) : null
+            }
+            sdsStyle="light"
+            arrow
+            placement="left"
+            enterNextDelay={700}
```

Comment:
Out of curiosity, what is the default value?

Reply:
If `enterNextDelay` isn't set then it just immediately spams the tooltip when you hover over the labels leaving tooltips overlapping each other until they disappear

The default for `enterNextDelay` is 0